ui/components/ui/contract-token-values/index.scss

```
@@ -0,0 +1,17 @@
+.contract-token-values {
+  &__copy-adress {
+    margin-inline-start: 10px;
```

Use 8px grid values here
```suggestion
    margin-inline-start: 8px;
```

---

@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).